docs/design/kata-guest-image-management-design.md

```diff
@@ -49,13 +49,57 @@ Pull the container image directly from the guest VM using `nydus snapshotter` ba
 #### Architecture
 
 The following diagram provides an overview of the architecture for pulling image in the guest with key components. 
-![guest-image-management-architecture](arch-images/guest-image-management-architecture.png)
+```mermaid
+flowchart LR
+    Kubelet[kubelet]--> |1 Pull image request & metadata|Containerd
+    Containerd-->Snapshotter[Nydus Snapshotter]
+    Snapshotter-->|2 pack image info| Containerd
+    Containerd-->Runtime[Kata Runtime]
+    Runtime-->Hypervisor
+    Hypervisor-->TEE
+    Runtime-->|3 pass image info to VM| Agent
+    E[Container Images Registry]-->|5 pull image with image info|CDH1
+    subgraph TEE [Virtual Machine]
+      Images[Container Images]-->|6 prepare container rootfs|H[Container]
+
+      subgraph CDH [Confidential Data Hub]
+        CDH1[Image Mgmt]
+      end
+
+      CDH-->Images
+      Agent[Kata Agent]-->|4 call image pull RPC|CDH
```

Comment:
nit: Add a period after each number and capitalise for readability
```suggestion
    Kubelet[kubelet]--> |1. Pull image request & metadata|Containerd
    Containerd-->Snapshotter[Nydus Snapshotter]
    Snapshotter-->|2. Pack image info| Containerd
    Containerd-->Runtime[Kata Runtime]
    Runtime-->Hypervisor
    Hypervisor-->TEE
    Runtime-->|3. Pass image info to VM| Agent
    E[Container Images Registry]-->|5. Pull image with image info|CDH1
    subgraph TEE [Virtual Machine]
      Images[Container Images]-->|6. Prepare container rootfs|H[Container]

      subgraph CDH [Confidential Data Hub]
        CDH1[Image Mgmt]
      end

      CDH-->Images
      Agent[Kata Agent]-->|4. Call image pull RPC|CDH
```

Reply:
The render of `mermaid` would meet some trouble when I used `.`. Let me try to resolve this.

Reply:
Ok, no problem if there is a mermaid issue. I think the capitalisation will help with the distinction

Reply:
Fixed, thanks